presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -88,12 +101,23 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+Hadoop catalog
+^^^^^^^^^^^^^^
+
+Iceberg connector supports Hadoop as Catalog
+
 .. code-block:: none
 
     connector.name=iceberg
-    iceberg.catalog.type=nessie
-    iceberg.catalog.warehouse=/tmp
-    iceberg.nessie.uri=https://localhost:19120/api/v1
+    hive.metastore.uri=hostname:port
+    iceberg.catalog.type=hadoop
+
+.. note::
+
+    To use Hadoop as catalog with Iceberg table requires the hive.metastore.uri property, even
+    though it doesn't rely on metastore for metadata when using Hadoop Catalog. You can provide
+    a placeholder with ``hive.metastore.uri=thrift://localhost:9083``. This will be fixed in a
```

Comment:
I have put up details about the issue in here https://github.com/prestodb/presto/issues/19920 
Currently since `hive.metastore.uri` property can not be null when using `Hadoop` as a catalog in Iceberg connector, we have to specify some placeholder for `hive.metastore.uri` property (doesn't necessarily have to be `thrift://localhost:9083`) which has to be fixed in this case since `Hadoop` catalog doesn't require any HMS for metadata.

Open for suggestions on how we want to add a note here about this, we can also add this issue link in the document until this is fixed?

Reply:
The Presto metadata store and Iceberg metadata store are two independent designs but they are coupled in the Presto-Iceberg connector, which, as a consequence, causes the dependency on the Hive metastore although the Iceberg Hadoop catalog doesn't use that. 
Adding an issue link here looks good to me. Or add it to the Section `Iceberg Connector Limitations`. Your call.